app/poros/json_schema_validator.rb

```diff
@@ -118,6 +108,12 @@ def copy_data_to_clipboard_and_open_schema_converter_if_development
         puts('Copied JSON to clipboard.'.yellow)
       end
 
+      # Open schema file in editor (creating it, if needed).
+      FileUtils.mkdir_p(File.dirname(absolute_schema_path))
+      FileUtils.touch(absolute_schema_path)
+      system("$EDITOR '#{absolute_schema_path}'", exception: true)
```

Comment:
I tried to open the editor before opening the converter in the browser, in the hope that the browser will then be "on top", but unfortunately VS Code is slow enough to open that it still ends up coming out "on top" at least some of the time. Not a huge deal, though, and I'm not sure there's much we can reasonably do to change this.